--- libpri-1.0.9/Makefile.orig	2005-05-11 03:37:29.000000000 +0000
+++ libpri-1.0.9/Makefile	2005-09-14 22:16:19.601647288 +0000
@@ -62,14 +62,14 @@
 	@cvs update -d
 
 install: $(STATIC_LIBRARY) $(DYNAMIC_LIBRARY)
-	mkdir -p $(INSTALL_PREFIX)/usr/lib
+	mkdir -p $(INSTALL_PREFIX)$(LIBDIR)
 	mkdir -p $(INSTALL_PREFIX)/usr/include
 	install -m 644 libpri.h $(INSTALL_PREFIX)/usr/include
-	install -m 755 $(DYNAMIC_LIBRARY) $(INSTALL_PREFIX)/usr/lib
-	if [ -x /usr/sbin/sestatus ] && ( /usr/sbin/sestatus | grep "SELinux status:" | grep -q "enabled"); then  restorecon -v $(INSTALL_PREFIX)/$(INSTALL_BASE)/lib/$(DYNAMIC_LIBRARY); fi
-	( cd $(INSTALL_PREFIX)/usr/lib ; ln -sf libpri.so.1 libpri.so )
-	install -m 644 $(STATIC_LIBRARY) $(INSTALL_PREFIX)/usr/lib
-	/sbin/ldconfig
+	install -m 755 $(DYNAMIC_LIBRARY) $(INSTALL_PREFIX)$(LIBDIR)
+#	if [ -x /usr/sbin/sestatus ] && ( /usr/sbin/sestatus | grep "SELinux status:" | grep -q "enabled"); then  restorecon -v $(INSTALL_PREFIX)/$(INSTALL_BASE)/lib/$(DYNAMIC_LIBRARY); fi
+	( cd $(INSTALL_PREFIX)$(LIBDIR) ; ln -sf libpri.so.1 libpri.so )
+	install -m 644 $(STATIC_LIBRARY) $(INSTALL_PREFIX)$(LIBDIR)
+#	/sbin/ldconfig
 
 uninstall:
 	@echo "Removing Libpri"
